README.md

```diff
@@ -52,6 +52,29 @@ The `get_trigger.ts` file is executed as a Deno program and takes one required a
 deno run -q --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_hooks/get_trigger.ts --source="./trigger.ts"
 ```
 
+### Generate Trigger Script Usage
+
+The `generate_trigger.ts` file is executed as a Deno program and takes one required argument:
+
+| Arguments  | Description                                           |
+| ---------- | ----------------------------------------------------- |
+| `--source` | JSON representation of trigger definitions. 
+               Expected format:  
+                    {
+                      "triggers": [{
+                        "type": "shortcut",
+                        "name": "test_reverse trigger",
+                        "description": "Reverses the string",
+                        "workflow": {"callback_id": "test_reverse"}
+                      }]
```

Comment:
Is there an `inputs` property here that is missing? Just comparing it to the example section below.